src/shared/modules/udc/udcDuck.ts

```diff
@@ -324,3 +325,13 @@ export const trackErrorFramesEpic: Epic<Action, GlobalState> = (
       }
     })
     .ignoreElements()
+
+export const trackPreviewEpic: Epic<Action, GlobalState> = action$ => {
+  return action$.ofType(PREVIEW_EVENT).map((action: any) => {
+    return metricsEvent({
+      category: 'preview',
+      label: action.label,
+      data: action.data
+    })
+  })
+}
```

Comment:
adding this epic for tracking preview events here so that we can consume them in the existing subscriber in the App and dispatch to segment